packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -35,6 +35,10 @@ export namespace WorkspaceCommands {
         id: 'workspace:open',
         label: 'Open...'
     };
+    export const OPEN_RECENT: Command = {
+        id: 'workspace:openRecent',
+        label: 'Open Recents Workspace...'
```

Comment:
*Open Recents Workspace* -> *Open Recent Workspace*

Reply:
Remove the "S" on Recent